scan-sonar.sh

```
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Set the directory containing your docker compose files
+DIR="docker"
+cd "$(dirname "$0")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)